200-CNAB-registries.md

```diff
@@ -15,3 +16,39 @@ This specification describes how CNAB bundles can be stored inside of OCI Regist
 This specification, the CNAB Registries specification, is not part of the CNAB Core specification. An implementation of CNAB Core MAY NOT implement this specification, yet still claim compliance with CNAB Core. A CNAB implementation MUST NOT claim to be CNAB Registries-compliant unless it meets this specification.
 
 > Previous versions of the CNAB Core draft specification included a repository protocol. That work is completely superseded by this specification.
+
+This specification is distinct from the CNAB Core specification. An implementation may comply with the CNAB Core specification, and yet not comply with this specification.
+
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt). The use of these keywords in this document are statements about how a CNAB implementation may fulfill the CNAB registry storage specification only.
+
+The CNAB Core 1.0 specification (CNAB1) does not dictate how bundles should be distributed. This is intentional, so that organizations that already have a way of distributing artifacts may continue to use it.
+This document, the CNAB Registries specification, is not part of the core specification, and an implementation of CNAB Core MAY NOT implement it, yet still claim compliance with CNAB Core. A CNAB implementation MUST NOT claim to be CNAB Registries compliant unless it meets this specification.
+
+This specification proposes the use of [OCI registries][oci-org] to distribute CNAB artifacts.
+
+## Approach
+
+<!--
```

Comment:
Maybe remove this comment block of notes to self?

Reply:
+1 We can create issues against the spec for things that haven't been defined yet like annotations, and storing images for air-gapped use.